src/libaktualizr/package_manager/dockerappmanager_test.cc

```diff
@@ -72,6 +72,7 @@ TEST(DockerAppManager, DockerApp_Fetch) {
   config.pacman.docker_apps_root = temp_dir / "docker_apps";
   config.pacman.docker_apps.push_back("app1");
   config.pacman.docker_app_bin = config.pacman.docker_compose_bin = "src/libaktualizr/package_manager/docker_fake.sh";
+  config.pacman.docker_prune_enabled = false;
```

Comment:
Didn't check if other bool related configs are also adding _enabled, but maybe just rename to 'docker_prune' instead? False would mean disabled already.